### PR TITLE
Release PR for 1.1.2

### PR DESCRIPTION
**Pull request by jfy133**
Includes docker hotfix

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/funcscan/tree/master/.github/CONTRIBUTING.md)
- [ ] If necessary, also make a PR on the nf-core/funcscan _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Review by louperelo — APPROVED** [review]
Go ahead

File: CHANGELOG.md [hunk 1]
```diff
@@ -3,6 +3,18 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v1.1.2 - [2023-06-29]
+
+### `Added`
+
+### `Fixed`
+
+- [#279](https://github.com/nf-core/funcscan/pull/279) Fix docker/podman registry definition for tower compatibility (fix by @adamrtalbot)
```
Comment (jasmezz) [inline CHANGELOG.md:12]:
```suggestion
- [#279](https://github.com/nf-core/funcscan/pull/279) Fix docker/podman registry definition for tower compatibility. (fix by @adamrtalbot)
```

**Review by jasmezz — APPROVED** [review]
:rocket: after checks pass

File: conf/base.config [hunk 2]
```diff
@@ -114,7 +114,8 @@ process {
 
     withName: DEEPARG_DOWNLOADDATA {
         memory = { check_max( 2.GB * task.attempt, 'memory'  ) }
-        cpus   = 1
+        cpus   = 1
```
Comment (jfy133) [inline conf/base.config:117]:
```suggestion
        cpus   = 1
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -3,6 +3,18 @@
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## v1.1.2 - [2023-06-29]
```
Comment (jfy133) [inline CHANGELOG.md:6]:
```suggestion
## v1.1.2 - [2023-06-30]
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -3,6 +3,18 @@
...
+
+### `Added`
+
+### `Fixed`
+
+- [#279](https://github.com/nf-core/funcscan/pull/279) Fix docker/podman registry definition for tower compatibility. (fix by @adamrtalbot)
```
Comment (jfy133) [inline CHANGELOG.md:12]:
```suggestion
- [#279](https://github.com/nf-core/funcscan/pull/279) Fix docker/podman registry definition for tower compatibility. (♥️ to sunitj for reporting, fix by @adamrtalbot)
```

**Pull request merged**